sktime/forecasting/compose/_stack.py

```diff
@@ -12,23 +12,39 @@
 
 from sktime.forecasting.base._base import DEFAULT_ALPHA
 from sktime.forecasting.base._meta import _HeterogenousEnsembleForecaster
-from sktime.forecasting.base._sktime import _RequiredForecastingHorizonMixin
 from sktime.forecasting.model_selection import SingleWindowSplitter
 
 from warnings import warn
 
 
-class StackingForecaster(
-    _RequiredForecastingHorizonMixin, _HeterogenousEnsembleForecaster
-):
+class StackingForecaster(_HeterogenousEnsembleForecaster):
+    """StackingForecaster.
+
+    Stacks two or more Forecasters
+
+    Parameters
+    ----------
+    forecasters : list of (str, estimator) tuples
+    final_regressor: Regressor
+    n_jobs : int or None, optional (default=None)
+        The number of jobs to run in parallel for fit. None means 1 unless
+        in a joblib.parallel_backend context.
+        -1 means using all processors.
+    """
+
     _required_parameters = ["forecasters", "final_regressor"]
+    _tags = {
+        "univariate-only": True,
+        "requires-fh-in-fit": True,
```

Comment:
hm, similar as for the `MultiplexForecaster` - does whether a `fh` is required in `fit` not depend on the components?

Reply:
ok, tags issue is now in #981.
For now, this is fine.